
sensor_msgs/CameraInfo camera_info
geometry_msgs/Transform local_transform

